### PR TITLE
Sync with `polkadot-v1.0.0` branch

### PR DESCRIPTION
**Pull request by ascjones**
- [x] Needs update of https://github.com/727-Ventures/pallet-assets-chain-extension to `polkadot-v1.0.0`
- [x] Diff and apply commits between `0.9.43` and `1.0.0`

### CODE REVIEW

**Review by athei — APPROVED** [review]

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -13,45 +13,45 @@ publish = false
...
-sp-transaction-pool = { git = "https://github.com/paritytech/substrate", package = "sp-transaction-pool", default-features = false, branch = "polkadot-v0.9.43" }
-sp-version = { git = "https://github.com/paritytech/substrate", package = "sp-version", default-features = false, branch = "polkadot-v0.9.43" }
-pallet-utility = { git = "https://github.com/paritytech/substrate", package = "pallet-utility", default-features = false, branch = "polkadot-v0.9.43" }
+pallet-authorship = { git = "https://github.com/paritytech/substrate", package = "pallet-authorship", default-features = false, branch = "polkadot-v1.0.0" }
+pallet-assets = { git = "https://github.com/paritytech/substrate", package = "pallet-assets", default-features = false, branch = "polkadot-v1.0.0" }
+pallet-balances = { git = "https://github.com/paritytech/substrate", package = "pallet-balances", default-features = false, branch = "polkadot-v1.0.0" }
+frame-support = { git = "https://github.com/paritytech/substrate", package = "frame-support", default-features = false, branch = "polkadot-v1.0.0" }
```
Comment (ggwpez) [inline runtime/Cargo.toml:22]: Can the runtime stay at .43? 1.0 was not a runtime release.

Comment (ggwpez) [reply runtime/Cargo.toml:22]: Okay it probably does not matter much here, as long as it compiles.

Comment (ascjones) [reply runtime/Cargo.toml:22]: Yes, we are just piggybacking on the whole `1.0` branch for our stable target for this node used in `ink` and `cargo-contract` development.

**Comment by ascjones** [comment]
> You forgot to update the README

https://github.com/paritytech/substrate-contracts-node/pull/196

**Pull request merged**